google/cloud/forseti/services/inventory/base/cai_gcp_client.py

```diff
@@ -52,23 +92,101 @@ def session(self):
         self._local.cai_session = db.create_readonly_session(engine=self.engine)
         return self._local.cai_session
 
-    def iter_compute_backendservices(self, project_number):
-        """Iterate Backend services from Cloud Asset data.
+    def fetch_billing_account_iam_policy(self, account_id):
+        """Gets IAM policy of a Billing Account from GCP API.
 
         Args:
-            project_number (str): id of the project to query.
+            account_id (str): id of the billing account to get policy.
+
+        Returns:
+            dict: Billing Account IAM policy.
+        """
+        resource = self.dao.fetch_cai_asset(
+            ContentTypes.iam_policy,
+            'google.cloud.billing.BillingAccount',
+            '//cloudbilling.googleapis.com/{}'.format(account_id),
+            self.session)
+        if resource:
+            return resource
+        # Fall back to live API if the data isn't in the CAI cache.
+        return super(CaiApiClientImpl, self).fetch_billing_account_iam_policy(
+            account_id)
+
+    def iter_billing_accounts(self):
+        """Iterate visible Billing Accounts in an organization from GCP API.
```

Comment:
This would not be `from GCP API`, based on the usage of `self.dao.iter_cai_assets()` only.